### ISSUE TITLE
logical_operations example doesn't work

### ISSUE THREAD

**Issue by ChrJLY** [Artelnics/OpenNN#59]
Hi,
I compiled successfully the library and the example apps under Linux Debian (g++ 6.3).
But the logical_operations application displays the wrong results:

Epoch 78: Minimum loss decrease (0) reached.
Loss decrease: 0
Parameters norm: 8.73467
Training loss: 2.43129e-31
Gradient norm: 3.83038e-16
Training rate: 9.0008e-18
Elapsed time: 00:00
X Y AND OR NAND NOR XOR XNOR
1 1 0 0 1 1 0 1
1 0 0 0 1 1 0 1
0 1 0 0 1 1 0 1
0 0 0 0 1 1 0 1

**Comment by Carlosbarranquero** [comment]
Hi,
logical_operations example has been updated.

**Comment by ChrJLY** [comment]
Thank you.
